README.md

```diff
@@ -194,10 +194,10 @@ can be useful for people who aren't as familiar with the app, or with certain
 complex operations that just aren't already packaged up in the app.
 
 In order to use the external environment functionality you must set the
-environments in `config/settings/development.local.yml` like so:  
+environments in `config/settings/development.local.yml`, `config/settings/qa.local.yml`
+`config/settings/staging.local.yml` and `config/settings/production.local.yml` like so:  
```

Comment:
We should be able to just add these to the file `config/settings.local.yml` instead of requiring them in all the environments.